### PR TITLE
Add German store IDs

### PR DESCRIPTION
**Pull request by muffix**
Adds a readme file with German Apple retail store IDs

### CODE REVIEW

**Comment by muffix** [comment]
This is going to require support querying the German store, similar to what #2 does for Australia and Canada. Happy to add it once #2 has been merged.
The SKUs in Germany appear to be different, too.

**Comment by worthbak** [comment]
nice - appreciate you pulling those store IDs together! #2 was just merged if you wanna take a look at adding Germany to the list of configurable countries?

**Comment by muffix** [comment]
Updated the PR. 

The SKUs all follow the same format: First 5 letters, then a country code, then `/A`. There's no need to list all of them for each country in the `constants.js` file. Instead, we can just have them be generated for the country in question. That should make it significantly easier to add support for more countries in the future. All we need to add is the path for the store and the country code of the SKU (`/de` and `D` for Germany, `/au` and `X` for Australia, etc).

**Comment by muffix** [comment]
Also added all Apple Retail stores sorted by city.

**Comment by muffix** [comment]
Wrong button, apologies. I didn't mean to close this. Can you have a look again @worthbak?

File: index.js [hunk 1]
```diff
@@ -1,43 +1,66 @@
...
-    skuList = SKUS_AUSTRALIA;
-    favorites = favoritesAustralia;
-  }
 }
 
+const countryConfig = COUNTRIES[country];
+
+let storePath = countryConfig["storePath"];
+let skuList = skusForCountry(countryConfig["skuCode"]);
+let favorites = favouritesForCountry(countryConfig["skuCode"]);
```
Comment (worthbak) [inline index.js:54]: just curious - why not use dot-syntax here? ex:
```js
let storePath = countryConfig.storePath;
let skuList = skusForCountry(countryConfig.skuCode);
let favorites = favouritesForCountry(countryConfig.skuCode);
```

**Review by worthbak — APPROVED** [review]
excellent work - thanks @muffix!! very cool insight into the model number variances between countries - I hadn't caught that difference. this will make it dramatically easier to fix issues with other countries (UK is next). merging 👍

File: index.js (hunk 1 above)
Comment (muffix) [reply index.js:54]: Mostly because I'm not a regular JS developer. 😅

Comment (worthbak) [reply index.js:54]: hahah that's mostly why i asked - I'm used to working in Swift, so I just assume others know more about JS than i do 😆

**Pull request merged**